<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scheduler</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource/res.qrc">
    <normaloff>:/ico/process_9539868.ico</normaloff>:/ico/process_9539868.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="staticbutt">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>630</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 10px;
border: 2px solid #1864ab;
border-radius: 6px;
background-color: #fff;
color:#333;
font-size:18px;
font-weight:bold;
</string>
    </property>
    <property name="text">
     <string>Static</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Schedular_type">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>130</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 10px;
border: 2px solid #1864ab;
border-radius: 6px;
background-color: #fff;
font-size:14px;
text-align:center;</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="maxCount">
     <number>2147483647</number>
    </property>
    <item>
     <property name="text">
      <string>FCFS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SJF_Preemptive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SJF_Non_Preemptive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Priority_Preemptive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Priority_Non_Preemptive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Round_Robin</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>136</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333;
font-weight: bold;
text-align: center;
font-size: 24px;</string>
    </property>
    <property name="text">
     <string>Choose a scheduler:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>61</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333;
font-weight: bold;
text-align: center;
font-size: 24px;</string>
    </property>
    <property name="text">
     <string> Number of processes:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>58</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 10px;
border: 2px solid #1864ab;
border-radius: 6px;
background-color: #fff;
font-size:14px;
text-align:center;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dynamicbutt">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>630</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 10px;
border: 2px solid #1864ab;
border-radius: 6px;
background-color: #1864ab;
color:#fff;
font-size:18px;
font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Dynamic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>455</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333;
font-weight: bold;
text-align: center;
font-size: 24px;</string>
    </property>
    <property name="text">
     <string>Time :</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>1104</x>
      <y>456</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#0d66b0;
font: 14pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_or_resume">
    <property name="geometry">
     <rect>
      <x>1299</x>
      <y>465</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
border: 2px solid #1864ab;
border-radius: 6px;
background-color: #fff;
color:#333;
font-size:14px;
font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="inputTable">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>279</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0d66b0;
color:#333;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <column>
     <property name="text">
      <string>Process id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Arrival time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Burst time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="quantam_label">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>183</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333;
font-weight: bold;
text-align: center;
font-size: 24px;</string>
    </property>
    <property name="text">
     <string>Quantam:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantam">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>196</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 2px;
padding-left: 10px;

border: 2px solid #1864ab;
border-radius: 6px;
background-color: #fff;
font: 9pt &quot;Segoe UI&quot;;
text-align:center;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="readyQeue">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>520</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#0d66b0;
color:#333;</string>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Process name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Remaining time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>State</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="turn_time">
    <property name="geometry">
     <rect>
      <x>1232</x>
      <y>40</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#333;
font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="waiting_time">
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>40</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color:#333;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>

   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333;
font-weight: bold;
text-align: center;
font-size: 24px;</string>
    </property>
    <property name="text">
     <string>Add new Process:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addedp">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>362</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 2px;
padding-left: 10px;

border: 2px solid #1864ab;
border-radius: 6px;
background-color: #fff;
font: 9pt &quot;Segoe UI&quot;;
text-align:center;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>362</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 5px;
border: 2px solid #1864ab;
border-radius: 6px;
background-color: #fff;
color:#333;
font-size:14px;
font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource/res.qrc"/>
  <include location="Resource/res.qrc"/>
 </resources>
 <connections/>
</ui>
